compiler-plugin/src/main/kotlin/arrow/meta/quotes/Transform.kt

```diff
@@ -106,5 +149,10 @@ sealed class Transform<out K : KtElement> {
 
     val empty: Transform<Nothing> = Empty
   }
-
 }
+
+operator fun <K : KtElement> Transform<K>.plus(transform: Transform<K>): Transform.Many<K> =
+  Transform.Many(arrayListOf(this, transform))
+
+operator fun <K : KtElement> Transform.Many<K>.plus(transform: Transform<K>): Transform.Many<K> =
+  Transform.Many(this.transforms.also { it.add(transform) })
```

Comment:
```suggestion
  Transform.Many(this.transforms.add(transform))
```
can we reduce it to this?

Reply:
ArrayList `add` function actually returns a boolean with the result of the addition, that's why I put it on an apply